src/global.d.ts

```diff
@@ -33,6 +33,7 @@ declare module 'clientConfig' {
     minChars?: number;
     coreName?: string;
     solrQueryConfig?: SolrQueryConfig;
+    featureResultZoomOffset?: number;
```

Comment:
Do we need to have this configurable? I would favour to have this detected automatically (if possible).

Reply:
Yes, that works too. Some other changes have been reversed and the files have been adjusted accordingly in [50832de](https://github.com/terrestris/shogun-gis-client/pull/1399/commits/50832de8f122b0301791808cfcb479659376d980).